ArduPlane/Parameters.cpp

```diff
@@ -1086,6 +1086,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Bitmask: 11: Disable suppression of fixed wing rate gains in ground mode
     // @Bitmask: 12: Enable FBWB style loiter altitude control
     // @Bitmask: 13: Indicate takeoff waiting for neutral rudder with flight control surfaces
+    // @Bitmask: 14: Climb the aircraft without doing slope in AUTO
```

Comment:
this description needs work....most dont know "slope" in regards to AUTO

Reply:
I added more description as you suggested